package.json

```diff
@@ -62,7 +62,8 @@
                 "command": "staticWebApps.createStaticWebApp",
                 "title": "%staticWebApps.createStaticWebApp%",
                 "category": "Azure Static Web Apps",
-                "icon": "$(add)"
+                "icon": "$(add)",
+                "enablement": "!virtualWorkspace"
```

Comment:
I think the "when" property needs to be updated as well. 
Here: 
```
"menus": {
            "view/title": [
                {
                    "command": "staticWebApps.createStaticWebApp",
                    "when": "view == staticWebApps",
                    "group": "navigation@1"
                },
```

Reply:
Using the "enablement" makes it disabled everywhere, (commands, keybindings, menus). Here is what it looks like on the ribbon:

![image](https://user-images.githubusercontent.com/12476526/130133482-7aeba283-2a09-4899-993c-ee5180a751aa.png)

My thinking was it might be confusing to users if the buttons they are accustomed to seeing on the ribbon are gone. So keeping them there, and disabling them might be better.

Thoughts?

Reply:
That's true, people might think it's a bug it they don't see it. I think it will be nice to have some notification pop-up for why it is disabled.

Reply:
I was thinking that too. A notification or even a popup or tooltip that appeared when you hover over the disabled button. Let me see if that's possible.

Reply:
Without creating a whole new command changing the tooltip isn't possible. And to display a notification we'd have to enable the button. 

I think this might be worth asking the VS Code team for guidance and see if they could add something to help out with this issue. I 100% think you're right in that users will probably not know why the buttons are disabled. 🤔

Reply:
When does this description appear?
![image](https://user-images.githubusercontent.com/17537419/130137311-3fc8b268-c4af-4b13-92b8-a3a648324cc1.png)

Reply:
I created https://github.com/microsoft/vscode/issues/131205 asking for a feature to help us with this.

Reply:
> When does this description appear?
![image](https://user-images.githubusercontent.com/17537419/130137311-3fc8b268-c4af-4b13-92b8-a3a648324cc1.png)

When in a virtual workspace, it appears here in this hover, and it appears on the extension page. 

![image](https://user-images.githubusercontent.com/12476526/130138043-2c9bf9a6-29ae-4abf-bfad-c39f261c3645.png)